### ISSUE TITLE
Enable Azure Blob Storage as backup path

### ISSUE THREAD

**Issue by WeyardWiz** [sqlcollaborative/dbatools#7763]
### Summarize Functionality

We are attempting logshipping using the `Invoke-DbaDbLogShipping` command.
Basically we want to store backups on Azure Blob storage, yet `Invoke-DbaDbLogShipping` requires a `$BackupNetworkPath`. The closest I was to this was Azure File Shares but unfortunately port 445 is blocked it seems so I cant utilize that.

Please create the ability to choose Azure Blob Storage as a backup/copy destination path instead of a network drive path. 

### Is there a command that is similiar or close to what you are looking for?

No

**Comment by potatoqualitee** [comment]
hey weyard! generally, sander takes care of this set of commands, i'll ping him and see whats up.

**Comment by WeyardWiz** [comment]
> hey weyard! generally, sander takes care of this set of commands, i'll ping him and see whats up.

Thanks Chrissy, any update from sander?

**Comment by potatoqualitee** [comment]
yes, sorry. he said it's a great idea but will take a lot of work and changes so may take a bit. @jpomfret -- I believe you recently wrote about this topic, any interest in adding some blog storage support?

**Comment by jpomfret** [comment]
Hey @potatoqualitee - I would love to, a little swamped with Grillen & PSConfEU at the moment, but I'll chat with Sander and see what we can do :)

**Comment by sanderstad** [comment]
@jpomfret Let's do that!

@WeyardWiz Sorry for not replying earlier but it's been pretty busy and this change is going to take some work but we'll work on it :)

**Comment by WeyardWiz** [comment]
> @jpomfret Let's do that!
> 
> @WeyardWiz Sorry for not replying earlier but it's been pretty busy and this change is going to take some work but we'll work on it :)

Awesome thanks, looking forward to it guys!

**Comment by WeyardWiz** [comment]
@sanderstad @potatoqualitee @jpomfret Hey guys, any update about this feature?

**Comment by andreasjordan** [comment]
@claude please have a look at this issue.

**Comment by andreasjordan** [comment]
That's good news, so I close this now.
